package.json

```diff
@@ -31,14 +31,17 @@
     "dotenv": "^8.0.0",
     "elasticsearch": "^15.0.0",
     "form-data": "^3.0.0",
+    "google-protobuf": "^3.0.0",
     "graphql": "^14.0.0",
+    "grpc": "^1.11.0",
     "kcors": "^2.2.1",
     "koa": "^2.5.0",
     "koa-bodyparser": "^4.0.0",
     "koa-passport": "^4.0.1",
     "koa-router": "^8.0.0",
     "koa-session2": "^2.0.0",
     "koa-static": "^5.0.0",
+    "md5": "^2.2.1",
```

Comment:
Actually we can use node's standard library instead of an old dependency:
https://odino.org/generating-the-md5-hash-of-a-string-in-nodejs/